docs/src/main/sphinx/routines/set.md

```diff
@@ -0,0 +1,31 @@
+# SET
+
+## Synopsis
+
+```text
+SET identifier [, ...]  EQ expression
+```
+
+## Description
+
+Use the `SET` statement in [SQL routines](/routines/introduction) to assign a
+value to one or more variables, referenced by comma-separated `identifier`s. The
```

Comment:
We currently only support assigning to a single variable. Multi-assignment is used to destructure a row by assigning its fields to the list of variables. It's actually a different syntax, requiring parenthesis:
```
SET (identifier, [, ...]) = expression

Reply:
Is that supported now or will that come in the future